camel-core/src/main/java/org/apache/camel/model/RouteDefinition.java

```diff
@@ -1167,6 +1186,19 @@ public OutputTypeDefinition getOutputType() {
         return this.outputType;
     }
 
+    public List<PropertyDefinition> getRouteProperties() {
+        return routeProperties;
+    }
+
+    /**
+     * Set route properties to use.
```

Comment:
Improve this javadoc a bit - I think it may be the javadoc that the tooling grabs for documentation in the XML DSL